Game.playground/Contents.swift

```diff
@@ -0,0 +1,111 @@
+struct Player{
+    var name: String
+    var damage: Int
+    var helf: Int
+}
+class Team {
+    var teamPlayers = [Player]()
+    var teamName: String
+    var generalDamage: Int
+    var generalHelth: Int
+    var inTheGame: Bool
+    init(name: String,pl1: Player,pl2: Player, pl3: Player,pl4: Player) {
+        self.teamName = name
+        self.teamPlayers.append(pl1)
+        self.teamPlayers.append(pl2)
+        self.teamPlayers.append(pl3)
+        self.teamPlayers.append(pl4)
```

Comment:
Так будет удобнее :)
```suggestion
        teamPlayers = [pl1, pl2, pl3, pl4]
```